fudge-ratio/src/main/webapp/index.html

```diff
@@ -2,11 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Fudge Ratio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <script src="script.js"></script>
+    <title>Fudge Ratio</title>
   </head>
-  <body>
-      <h1> Hello world!! Start files finished :-) </h1>
-      <p>Awesome!</p>
+  <body class="container">
+    <h1> Fudge Ratio Calculator </h1>
+    <p>Click the start button when you're ready!</p>
+    <br/><br/>
+    <button onclick="startTimer()">Start</button>
```

Comment:
This is great, and is a perfectly valid option, but why clutter the HTML (given that HTML should be semantic and not mixed with styles and logic)?
Another option is to give the 2 buttons ID and the add the events listeners in Javascript

Reply:
I did it this way based on the tutorials I saw in the sps walkthroughs. I'll try using event listeners.